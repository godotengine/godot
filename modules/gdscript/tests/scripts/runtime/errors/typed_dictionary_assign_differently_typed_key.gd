func get_key() -> Variant:
	return "key"

func test():
	var typed: Dictionary[int, int]
	typed[get_key()] = 0
	print('not ok')
