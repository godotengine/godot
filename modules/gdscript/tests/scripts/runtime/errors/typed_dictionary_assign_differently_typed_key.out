GDTEST_RUNTIME_ERROR
>> SCRIPT ERROR
>> on function: test()
>> runtime/errors/typed_dictionary_assign_differently_typed_key.gd
>> 6
>> Invalid assignment of property or key 'key' with value of type 'int' on a base object of type 'Dictionary[int, int]'.
